validator/src/main/java/com/amazon/aoc/validators/AlarmPullingValidator.java

```diff
@@ -72,10 +72,14 @@ public void validate() throws Exception {
             log.info(metricAlarm.getStateValue());
             if (!metricAlarm.getStateValue().equals("OK")) {
               log.error(
-                  "alarm {} is alarming, status is {}, metric is {}, failing to bake",
+                  "alarm {} is alarming, status is {}, " 
```

Comment:
The format is due to the linter. 
+ must be a on a new line 
Must be under 100 char